README.md

```diff
@@ -1,17 +1,75 @@
 # Fleet Scheduler
 
-This service is responsible for executing jobs/tasks for Fleet
+Fleet-Scheduler is a collection of tasks that can each be set up as jobs in order to automate many things.
 
-## Usage
+A **task** is a set of functions that are executed in order to make changes to fleet services.
 
+A **job** is a task that is set up to be executed on a schedule.
+
+## Standalone Usage
+
+```
+./fleet-scheduler <task> --config ./path/to/config.yaml
 ```
-./fleet-scheduler <task>
+## Usage within sandbox
+
+Fleet Scheduler jobs can be created within the value.yaml file of the [sandbox](https://github.com/metal-toolbox/sandbox).
+
+Each job requires a few values in order to function.
+
+## Creating new tasks
+
+- Tasks need to be implemented in code within [Fleet-Scheduler](https://github.com/metal-toolbox/fleet-scheduler)
+- Tasks are implemented as [cobra](https://github.com/spf13/cobra) command line commands within /cmd
+- Take a look at /cmd/inventory.go for a good example.
+
+## Creating new jobs
+
+You just need to add the job to the [value.yaml](https://github.com/metal-toolbox/sandbox) file.
+Each job requires a few values in order to function.
+
+### Values for creating new jobs
+
+Job Values required and location to explain them with `kubectl explain <field>`
```

Comment:
I wonder if this section is better served from the doc in sandbox, internal k8s deployments docs.
Since thats where it can be kept updated and relevant, and it can be linked to from this README.

Reply:
Mhmm. So i put this doc in fleet-docs, delivery-infrastructure, and here. I think we would want some "source of truth" on these docs. What do we prefer?

Reply:
Resolved:
https://github.com/metal-toolbox/sandbox/pull/58